server/server.js

```diff
@@ -5,6 +5,24 @@ const path = require('path');
 
 const app = express();
 
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
Should we extract this into a development module that we just require as a line in the beginning?